apps/charterafrica/src/payload/blocks/Partners.js

```diff
@@ -0,0 +1,34 @@
+const Resources = {
+  slug: "block-partners",
+  labels: {
+    singular: {
+      en: "Partner",
+      fr: "Associé",
+      pt: "Parceiro",
+    },
+    plural: {
+      en: "Partners",
+      fr: "Les partenaires",
+      pt: "Parceiros",
+    },
+  },
+  fields: [
+    {
```

Comment:
Looking at

![Screenshot from 2023-01-23 15-49-34](https://user-images.githubusercontent.com/1779590/214044101-35b8ebb4-ee40-4971-8730-e0527ed23eec.png)

we need title hire i.e. Partners and may be optional description (though that may be just a design issue).

Reply:
Added